tests/xlsx/test_excel_writer.py

```diff
@@ -129,3 +123,182 @@ def test_write_xlsx__correctly_handles_custom_twitch_widths(mocker):
 
         # switch dir back to avoid causing issues with other tests
         os.chdir(cwd)
+
+
+@pytest.mark.parametrize("test_value", [None, True, False])
+def test_include_stim_protocols_runs_without_error(mocker, test_value):
```

Comment:
This and the tests below don't actually test any of the functionality provided the params they're supposed to be testing. In other words, they're basically just testing the params exist for `write_xlsx`. You could strip out all the functionality of `include_stim_protocols` and this test wouldn't fail since it's not actually making assertions on anything other than the output file name that gets returned. I know I mentioned before that you can typically pull from tests already written when writing new ones as they usually have some things in common, but the `test_write_xlsx__runs_without_error` test you copied is not a test that you can do that with (notice the comment: `# Tanner (10/4/22): do not add anything to this test, it is just meant to run a full analysis start to finish. Any and all param testing should be done in separate tests as is done below`). What you'll need to do is look at tests like `test_write_xlsx__correctly_handles_default_twitch_widths` and `test_write_xlsx__correctly_handles_custom_twitch_widths` that actually make assertions on the contents of the xlsx output and see how you can do something similar to test the functionality provided by these params